code/network/multi_log.cpp

```diff
@@ -77,11 +81,36 @@ void multi_log_write_update()
 	ml_printf("Server has been active for %d hours, %d minutes, and %d seconds", hours, mins, seconds);
 }
 
+// write out some info helpful for debugging
+static void multi_log_write_info()
+{
+	extern bool Multi_cfg_missing;
+
+	if (Multi_cfg_missing) {
+		ml_string("**  multi.cfg is missing!  **");
```

Comment:
Oh, so `ml_string` does a `\n` by default?

Reply:
Yep, the base generic logging function puts an EOL on every line by default. So both `ml_printf()` and `ml_string()` get `\n` automatically.